synapse/storage/schema/delta/55/add_spans_to_device_lists.sql

```diff
@@ -0,0 +1,20 @@
+/* Copyright 2019 The Matrix.org Foundation C.I.C
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Opentracing context data for inclusion in the device_list_update EDUs, as a
+ * json-encoded dictionary
+ */
+ALTER TABLE device_lists_outbound_pokes ADD context TEXT;
```

Comment:
can we call it `opentracing_context` (or even `ot_context` or something)?

Reply:
As discussed here: https://github.com/matrix-org/synapse/pull/5852#discussion_r315596130
This was also mentioned in the synapse-dev a few days ago.